Networking/Networking.xcodeproj/project.pbxproj

```diff
@@ -666,16 +667,6 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
```

Comment:
This seems to be done automatically by Xcode, I'm assuming because the 3 libraries added to the `Networking` target are set to `Do Not Embed`? I tested this by:

- If I switch `WooFoundation.framework` to `Embed & Sign` without building the app it only adds back the following sample lines
```
68FBC5C42893CEB500A05461 /* WooFoundation.framework in Frameworks */
68FBC5C42893CEB500A05461 /* WooFoundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 68FBC5B728928C8C00A05461 /* WooFoundation.framework */; };
```
- Now, after making this change I attempt to build the app (it actually fails to build because of the `Do Not Embed` problem), adds the parts we're missing in `PBXNativeTarget` and `PBXCopyFilesBuildPhase` sections within the `.pbxproj` file:
```
		68FBC5C62893CEB500A05461 /* Embed Frameworks */ = {
			isa = PBXCopyFilesBuildPhase;
			buildActionMask = 2147483647;
			dstPath = "";
			dstSubfolderSpec = 10;
			files = (
				68FBC5C52893CEB500A05461 /* WooFoundation.framework in Embed Frameworks */,
			);
			name = "Embed Frameworks";
			runOnlyForDeploymentPostprocessing = 0;
		};
68FBC5C62893CEB500A05461 /* Embed Frameworks */,
```
- If I switch the library back to `Do Not Embed`, the app builds but the code related to `PBXNativeTarget` and `PBXCopyFilesBuildPhase` disappears again from the file.

Reply:
Great @iamgabrielma , thanks for clarifying this. 
It seems like we had this build phase in the Networking project, which actually was not doing anything:

<img width="929" alt="Screenshot 2022-07-29 at 10 44 17" src="https://user-images.githubusercontent.com/1864060/181720917-f70ee23b-94bf-4e26-9369-368afa8ce3b7.png">

It was probably a remnant from other PR, or something generated when we had some embedded frameworks. Now that it is not the case, we can indeed get rid of it 👍